roles/ceph-docker-common/defaults/main.yml

```diff
@@ -2,3 +2,7 @@
 ceph_conf_overrides:
   global:
     log_file: /dev/null
+
+docker_pull_retry: 3
```

Comment:
These options should be in ceph-defaults btw, sorry to discover that late. Somewhere in here: https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-defaults/defaults/main.yml#L454

Once done, execute generate group vars.
After this, we should be okay.

Reply:
Why is it still here? This is not needed: `roles/ceph-defaults/defaults/main.yml` is enough.